poandy/controller/trade.py

```diff
@@ -4,7 +4,15 @@
 
 class TradeController(Controller):
     @classmethod
-    def get_trades(cls, account_id, state="OPEN", instrument=None, count=50):
+    def get_trades(
+        cls,
+        account_id,
+        ids=None,
+        state="OPEN",
+        instrument=None,
+        count=50,
+        beforeID=None,
+    ):
         if state not in ["OPEN", "CLOSED", "CLOSE_WHEN_TRADEABLE", "ALL"]:
```

Comment:
@callistachang @gau-nernst should we perform input validation at all or leave it to the api to do it? It feels a bit inconsistent that we are validating some inputs (like state) and not validating others, like tradeSpecifier

Reply:
i'm okay with not validating input. it would be good but this would only be most likely used by strat teams only so hopefully we can get this out to them asap.